region/maridia/inner-pink.json

```diff
@@ -1409,62 +846,36 @@
                 {"or": [
                   {"and": [
                     "canSwim",
+                    "Morph",
                     {"or": [
                       "canPassBombPassages",
                       "canUseSpringBall",
                       "can2HighWallMidAirMorph",
-                      {"and": [
-                        "canSuitlessMaridia",
-                        "Morph"
-                      ]}
+                      "canSuitlessMaridia"
```

Comment:
Is this referring to removing ones grav suit to make the jump easier?

Reply:
Yeah, that makes the mid-air morph viable.